packages/storybook/stories/Modal.stories.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import Modal from '../../react-components/src/components/Modal/Modal';
+import { Modal } from '@department-of-veterans-affairs/component-library';
```

Comment:
Would all of the imports in vets-website need to be updated to look like this as well?

Reply:
This is optional now that I've reverted the changes where the React components were no longer copied.